frontend/src/components/burger.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+function Burger() {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleClick = () => {
+    setIsActive(!isActive);
+    const navigationElement = document.querySelector('.navigation');
+    if (navigationElement) {
+      navigationElement.classList.toggle('navigation_active');
+    }
+  };
+
+  return (
+    <div className="burgerContainer">
+      <svg
```

Comment:
<img className='logo' src="/perfguardian-text-and-logo.svg" alt='perfguardian-text-and-logo' />
 rajouter un filtre pour le changement de couleur => darkmode